radixdlt/src/main/java/com/radixdlt/consensus/ValidatingEventCoordinator.java

```diff
@@ -106,16 +106,13 @@ private void startQuorumNewView(View view) {
 	}
 
 	private void proceedToView(View nextView) {
-		try {
-			// TODO make signing more robust by including author in signed hash
-			ECDSASignature signature = this.selfKey.sign(Hash.hash256(Longs.toByteArray(nextView.number())));
-			NewView newView = new NewView(selfKey.getPublicKey(), nextView, this.vertexStore.getHighestQC(), signature);
-			ECPublicKey nextLeader = this.proposerElection.getProposer(nextView);
-			log.info(String.format("%s: Sending NEW_VIEW to %s: %s", this.getShortName(), this.getShortName(nextLeader.getUID()), newView));
-			this.networkSender.sendNewView(newView, nextLeader.getUID());
-		} catch (CryptoException e) {
-			throw new IllegalStateException("Failed to sign new view", e);
-		}
+        // TODO make signing more robust by including author in signed hash
+		ECDSASignature signature = this.selfKey.sign(Hash.hash256(Longs.toByteArray(nextView.number())));
```

Comment:
Weird formatting

Reply:
@flotothemoon yeah strange, I enabled IntelliJ to render leading whitespaces and saw the discrepancy, I believe it is fixed now.